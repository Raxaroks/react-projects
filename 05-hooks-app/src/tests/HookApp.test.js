import '@testing-library/jest-dom';
import { shallow } from 'enzyme';
import HookApp from '../HookApp'; // componente a testear

describe('Pruebas en el componente: HookApp', () => {

    test('Debe de renderizarse correctamente.', () => {
        const componentWrapper = shallow(<HookApp/>);
        expect(componentWrapper).toMatchSnapshot();
    });
        
});
