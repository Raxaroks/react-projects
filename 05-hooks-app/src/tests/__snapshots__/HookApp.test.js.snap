// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el componente: HookApp Debe de renderizarse correctamente. 1`] = `
<div>
  <h1>
    Hola, mundo!
  </h1>
</div>
`;
